examples/datepicker/datepicker.dateValue.tsx

```diff
@@ -0,0 +1,54 @@
+import { type CodeData } from '~/components/code-demo';
+import { Datepicker } from '~/src';
+
+const code = `
+'use client';
+
+import { Datepicker } from 'flowbite-react';
+
+function Component() {
+  return (
+    <Datepicker
+      dateValue={new Date(2022, 11, 25)} // 25th December 2022
+    />
+  );
+}
+`;
+
+const codeRSC = `
+import { Datepicker } from 'flowbite-react';
+
+function Component() {
+  return (
+    <Datepicker
+      weekStart={new Date(2022, 11, 25);} // 25th December 2022
```

Comment:
instead of `weekStart` it should be `dateValue` maybe?  🤔

Reply:
You're correct, thank you @SutuSebastian.